installer/bundle/bundle_skel.sh

```diff
@@ -770,6 +843,8 @@ install_extra_package()
 ulinux_detect_installer
 set -e
```

Comment:
Move to after install_troubleshooter

Reply:
Fixed here:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/bff7c80f869d6f5a4046ed41b4bd0ed89d69219f/installer/bundle/bundle_skel.sh#L846-L849